### PR TITLE
More PHPStan fixes

### PR DESCRIPTION
**Pull request by Jean85**

### CODE REVIEW

**Comment by iansltx** [comment]
@Jean85 May have been due to the merge, but some tests here don't pass. Mind getting those tests to pass? Will merge thereafter.

**Comment by Jean85** [comment]
@iansltx I've rebased on top of your MR and fixed what was needed! :+1: tests should be green now
~I'll see if I can add more fixes too.~

File: src/Controller/EventHostsController.php [hunk 1]
```diff
@@ -48,7 +43,7 @@ public function listHosts(Request $request, PDO $db)
             throw new Exception('Event not found', Http::NOT_FOUND);
         }
 
-        $list = $mapper->getHostsByEventId($event_id, $verbose, $start, $resultsperpage);
+        $list = $mapper->getHostsByEventId($event_id, $resultsperpage, $start, $verbose);
```
Comment (iansltx) [inline src/Controller/EventHostsController.php:46]: Nice catch!

Comment (Jean85) [reply src/Controller/EventHostsController.php:46]: Well, that was thanks to the type mismatch! :smile:

**Pull request merged**